src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -301,11 +303,15 @@ function LoadoutBuilder({
     [statOrder, enabledStats, sets]
   );
 
-  const shareBuild = () => {
-    const urlParams = new URLSearchParams({
+  const shareBuild = (notes?: string) => {
+    const p: Record<string, string> = {
       class: classType.toString(),
       p: JSON.stringify(params),
-    });
+    };
+    if (notes) {
+      p.n = notes;
```

Comment:
Are you using single char params because you are concerned about hitting url length limits?

Reply:
Not really, just felt like it.